RubeesEat/Pages/Index.cshtml

```diff
@@ -3,34 +3,46 @@
-    var displayClass = isPositive ? "positive-amount" : "negative-amount";
-}
+<div class="container mt-3">
+    <h1 class="text-center">Hello, @personRepository.GetCurrentUser().FirstName!</h1>
 
-<span id="balance" t-id="balance" class="@displayClass">@amountAsString</span>
-
-<ul class="entryLines">
-    @foreach (var item in billRepository.GetRecentBalanceChanges(personRepository.GetCurrentUser(), 1, 5))
-    {
-        <li class = "@(item.Amount < 0 ? "negative-amount" : "positive-amount")">
-            <a class="clickBillDetails" t-action="showBillDetails" href="BillDetails/@item.BillId">
-                <div class="amount-description">
-                    <span class="amount @(item.Amount < 0 ? "negative-amount" : "positive-amount")" t-id="amount">
-                        @item.Amount.ToString("C", CultureInfo.CreateSpecificCulture("de-DE"))
-                    </span>
-                    <span class="description" t-id="description">
-                        @item.Description
-                    </span>
-                    <span class="date" t-id="date">
-                        @item.Date.ToString("D", CultureInfo.CreateSpecificCulture("en-US"))
-                    </span>
-                </div>
-            </a>
-        </li>
+    @{
+        var amount = billRepository.GetBalance(personRepository.GetCurrentUser());
+        var isPositive = amount >= 0;
+        var amountAsString = amount.ToString("C", CultureInfo.CreateSpecificCulture("de-DE"));
+        var displayClass = isPositive ? "text-success" : "text-danger";
     }
-</ul>
+
+    <div class="text-center mt-5">
+        <span t-id="balance" class="p-3 border rounded @displayClass">@amountAsString</span>
+    </div>
+
+    <ul class="entryLines list-group mt-5 mx-auto" style="max-width: 900px;">
+        @foreach (var item in billRepository.GetRecentBalanceChanges(personRepository.GetCurrentUser(), 1, 5))
+        {
+        <li class="list-group-item justify-content-between">
+                <a class="text-decoration-none text-dark w-100" t-action="showBillDetails" href="BillDetails/@item.BillId">
+                    <div class="amount-description">
+                        <div class="row">
+                            <div class="col-3 @(item.Amount < 0 ? "text-danger" : "text-success")" t-id="amount">
+                                @item.Amount.ToString("C", CultureInfo.CreateSpecificCulture("de-DE"))
+                            </div>
+                            <div class="col-5" t-id="description">
+                                @item.Description
+                            </div>
+                            <div class="col-4 text-end fst-italic" t-id="date">
+                                @item.Date.ToString("D", CultureInfo.CreateSpecificCulture("en-US"))
+                            </div>
+                        </div>
+                    </div>
+                </a>
+            </li>
```

Comment:
Right now, the elements are clickable but we don't get hover feedback for the elements, which is strange for the user. As a user I expect stuff that I can click to change in some way when I hover. We can fix this by dropping the `<ul>` and making the `<a>` tags to our list element items. There are examples of that on the bootstrap website